share/spice/statista/statista.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    
+    function getImage(item, size) {  
+        if (item.teaserImageUrls[size]) {
+            return item.teaserImageUrls[size].src;
+        } else {
+            return 'https://da73v1deq9k1l.cloudfront.net/Statistic/table/table-100-1.png';
```

Comment:
If there's no relevant icon to show, I think it's best we don't show anything at all so there's more space for the text, which is most important.